lib/pure/math.nim

```diff
@@ -56,7 +56,7 @@ type
     fcNegInf     ## value is negative infinity
 
 proc classify*(x: float): FloatClass =
-  ## classifies a floating point value. Returns `x`'s class as specified by
+  ## Classifies a floating point value. Returns `x`'s class as specified by
```

Comment:
I wasn't capitalizing for grammar. I was capitalizing because it reminded
me of a newspaper headline, which would be capitalized. The cases which use
complete sentences (usually longer with examples) obviously capitalize, and
I thought it looked a little sloppy to have both show up when their
formatting (starts on a new line) was the same.
I'm up for either, though, and I'm fine changing them to suit the crowd.
What's the stance on final periods?
On Jun 30, 2015 4:30 AM, "Andreas Rumpf" <notifications@github.com> wrote:

> In lib/pure/math.nim
> <https://github.com/nim-lang/Nim/pull/3021#discussion_r33550733>:
>
> > @@ -56,7 +56,7 @@ type
> >      fcNegInf     ## value is negative infinity
> >
> >  proc classify*(x: float): FloatClass =
> > -  ## classifies a floating point value. Returns `x`'s class as specified by
> > +  ## Classifies a floating point value. Returns `x`'s class as specified by
>
> So just document that both are fine but Uppercase is stupid and violates
> English grammar.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nim-lang/Nim/pull/3021/files#r33550733>.
>